hash/apple2gs_flop_orig.xml

```diff
@@ -3603,13 +3602,12 @@ license:CC0-1.0
 		<description>Dragon's Lair</description>
 		<year>2022</year>
 		<publisher>Brutal Deluxe Software</publisher>
-		<info name="release" value="2022-3-30"/>
+		<info name="usage" value="System Requirements: Apple IIgs ROM 01/3 with 1.5 MB RAM, one 3.5 disk drive, System 5.0.4. Recommended: Apple IIgs ROM 01/3 with 4 MB RAM, accelerator card, hard drive, System 6.0.1."/>		
```

Comment:
(Dragon's Lair, like 3606) There’s trailing space on this line that will cause churn.